pom.xml

```diff
@@ -128,7 +128,10 @@
     <ning.async.version>1.8.12</ning.async.version>
     <snappy.version>0.3</snappy.version>
     <sentry.version>6.0.0</sentry.version>
-    <horizon.version>0.0.20</horizon.version>
+    <horizon.version>0.0.25</horizon.version>
+    <dep.algebra.version>1.2</dep.algebra.version>
+    <dep.immutables-hubspot.version>1.2</dep.immutables-hubspot.version>
```

Comment:
I think `hubspot-immutables` is a better name (matches the repo name).

Reply:
👌